v3.0_roadmap.adoc

```diff
@@ -0,0 +1,233 @@
+= Road map for James 3.0
+
+This document aims at dividing existing and upcoming code into the following categories :
+
+ - supported
+ - experimental
+ - unsupported
+
+== Mailbox section
+
+.Feature matrix : mailbox backend
+|===
+|Feature name |Status | Comment
+
+|JPA
+|supported
+|Already existing code
+
+|Memory
+|supported
```

Comment:
I think it should be unsupported. Supported is for production purpose, we can't offer any guarantee when using a memory data storage IMO.